man/systemd-cryptenroll.xml

```diff
@@ -237,7 +237,12 @@
         <listitem><para>Configures the TPM2 PCRs (Platform Configuration Registers) to bind the enrollment
         requested via <option>--tpm2-device=</option> to. Takes a <literal>+</literal> separated list of
         numeric PCR indexes in the range 0…23. If not used, defaults to PCR 7 only. If an empty string is
-        specified, binds the enrollment to no PCRs at all. PCRs allow binding the enrollment to specific
+        specified, binds the enrollment to no PCRs at all.
+        Users may also specify the specific registers they want to bind the enrollment to using string aliases
```

Comment:
"Users may also specify the specific registers they want to bind the enrollment to" → "Registers may also be specified"

The part starting with "for instance" should be a new sentence.